tests/functional_tests/mining.py

```diff
@@ -71,6 +75,12 @@ def create(self):
     def mine(self, via_daemon):
         print("Test mining via " + ("daemon" if via_daemon else "wallet"))
 
+        time_pi_single_cpu = self.measure_cpu_power_get_time(1)
+        time_pi_all_cores = self.measure_cpu_power_get_time(multiprocessing.cpu_count())
```

Comment:
Would suggest putting the core count in a variable, as you use it repeatedly in calculations.

Reply:
Done. 

Regarding nproc, this variable name would suggest the source and quantity of the variable, but not the intention how it's meant to be used.

Reply:
All I will say here is that if you find `nproc` insufficient to document intent, you can always alias it while still avoiding repeatedly querying the machine's core count.

Reply:
The suggestion was correct, and I query the cpu count only once now (well twice, since the method is being called twice)